tools/harness-thci/ARM.py

```diff
@@ -2212,7 +2214,7 @@ def MGMT_ACTIVE_SET(self, sAddr='', xCommissioningSessionId=None, listActiveTime
                 cmd += str(xPanId)
 
             if xDelayTimer != None:
-                cmd += ' delay '
+                cmd += ' delaytimer '
```

Comment:
Delay Timer TLV should only appears in pending operational dataset, do we need to remove this from MGMT_ACTIVE_SET command?